app/javascript/packs/colorPreview.js

```diff
@@ -1,52 +1,47 @@
+/* eslint-disable no-param-reassign */
```

Comment:
this should be removed IMO. You should disable it locally with something like:

```javascript
function updateFields(field, targetField) {
  /* eslint-disable no-param-reassign */
  field.bgColor.value = targetField.bgColor.value;
  field.textColor.value = targetField.textColor.value;
  /* eslint-enable no-param-reassign */
}
```

Reply:
ok